src/components/10-atoms/checkbox/index.react.d.ts

```diff
@@ -8,6 +8,8 @@ interface AXACheckboxProps {
   disabled?: boolean;
```

Comment:
AFAIK, the lowercase types belong to javascript and the uppercase ones to typescript.

Reply:
no, no.